docs/getting-started.md

```diff
@@ -134,7 +136,7 @@ RUN ../ostree-git.sh
 
 # builds ostree + any additional commands
 COPY ostree-test.sh /
-RUN ../ostree-test.sh
+RUN cd ostree && ../ostree-test.sh
```

Comment:
After `ostree-git.sh`, the Dockerfile commands still execute from the directory `/`, and the git commands in `ostree-test.sh` expect to be in the `ostree` repo. So a `cd ostree` is needed somewhere before the git commands in `ostree-test.sh`, but I think it should be outside of that script because we want to run the script from the `/ostree` (see [this](https://github.com/ostreedev/ostree/pull/1694/files/b6dc162c239c701beb89377a341370c93db87e1b..8f99b354750e6b8fdffe8d4b4a5c05f6761f5222#diff-a39c9f7aa728d5fa3b973bc6ba49228aR245)).

Just noticed at `/`, `.` and `..` are the same...

Reply:
Oh I could just move `WORKDIR /ostree` on top of  `RUN ../ostree-test.sh`. I didn't notice this. Thanks for pointing it out.